doc/user/datamover.md

```diff
@@ -0,0 +1,84 @@
+# Data Mover
+
+The Dataset Mover is a collection of multiple tools that allow users to copy
+and serialize data across DAOS and POSIX file systems. There is support for
+data movement across POSIX, DAOS, and HDF5 containers, as well as support for
+serializing a DAOS container to an HDF5 file and deserializing back to DAOS.
+
```

Comment:
@daltonbohning LGTM. The only thing that I think might be good to add here is a description of what serialization means, since it may not be clear to people how it is defined. You could add the description from the Milestone report: 
"DAOS Serialization is defined by being able to store a representation of the DAOS Container on a POSIX filesystem.  The serialized container is only for storage and can only be interacted with again by deserializing or importing the data back into a DAOS system. The serialization format used for DAOS is HDF5, where one container is serialized into HDF5 file(s).  A deserialization tool is used to restore a DAOS container from the HDF5 file(s). "

Maybe all of that doesn't need to be there, but having a sentence that describes how serialization is defined I think would be helpful.

Reply:
@dsikich How about this latest edit?

Reply:
> @dsikich How about this latest edit?

👍